client/my-sites/email/email-management/titan-management-nav/index.jsx

```diff
@@ -87,6 +99,59 @@ class TitanManagementNav extends React.Component {
 		);
 	};
 
+	renderMailboxSetupPrompt = () => {
+		const { currentRoute, domain, selectedSiteSlug, translate } = this.props;
+
+		const numberOfUnusedMailboxes =
+			getMaxTitanMailboxCount( domain ) - getConfiguredTitanMailboxCount( domain );
+
+		if ( numberOfUnusedMailboxes <= 0 ) {
+			return;
+		}
+
+		const showExternalControlPanelLink = ! isEnabled( 'titan/iframe-control-panel' );
+		const controlPanelUrl = showExternalControlPanelLink
+			? emailManagementTitanControlPanelRedirect( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } )
+			: emailManagementManageTitanAccount( selectedSiteSlug, domain.name, currentRoute, {
+					context: TITAN_CONTROL_PANEL_CONTEXT_CREATE_EMAIL,
+			  } );
+
+		return (
+			<CompactCard className="titan-management-nav__mailbox-setup-prompt">
+				<span>
+					<Gridicon icon="info-outline" size={ 18 } />
+					<em>
+						{ translate(
+							'You have %(numberOfMailboxes)d unused mailbox',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'You still have %(unusedMailboxCount)d unused mailbox', 'You still have %(unusedMailboxCount)d unused mailboxes' )` *ES Score: 6*